static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -1,15 +1,27 @@
 @import '../_vars';
 @import '../_mixins';
-
 $labour-liverpool-header-element-opacity: .85;
 
+// make bottom left corner of headline flush with the bottom of the main media image
+$labour-liverpool-headline-top-supports-cover-mobile: 319px;
+$labour-liverpool-headline-top-supports-cover-tablet: 314px;
+$labour-liverpool-headline-top: 250px; // IE / Edge
+
 .immersive-header-container--labour-liverpool {
-    min-height: gs-height(6);
+    min-height: gs-height(8);
 
     .immersive-main-media__media {
-        // fallback for IE not supporting object-fit: cover
-        @include mq($from: desktop) {
-            max-height: gs-height(8);
+        position: absolute;
+        width: 100%;
+        // Remove inline spacing
+        display: block;
+        height: gs-height(8);
+
+        @supports (object-fit: cover) and (height: 1vh) and (display: flex) {
```

Comment:
Most of it is. We'll still need the `height` to override the fallback height on line 18, but the rest can go.